apps/openmw/mwmechanics/aiwander.hpp

```diff
@@ -151,6 +154,9 @@ namespace MWMechanics
             static const std::string sIdleSelectToGroupName[GroupIndex_MaxIdle - GroupIndex_MinIdle + 1];
 
             static int OffsetToPreventOvercrowding();
+
+            //Idle animations that when called cause errors
+            std::vector<unsigned short> bad_idles;
```

Comment:
Two problems:
* Use the mBadIdles notation for member variables.
* The AiPackage should not contain run-time state, just basic information about the package. This change as is would increase the memory usage for every actor in the game (even in inactive cells). Runtime state should be added to AiWanderStorage.